### PR TITLE
doc: add minutes for meeting 4 Jan 2023

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by ovflowd — APPROVED** [review]
LGTM!

File: meetings/2023-01-04.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* OpenJS track at Open Source Summit in Vancouver 10-12 in May, planning collaborator summit May 9th, the day before the summit.
+
+### nodejs/next-10
+
+* Metadata Proposal for Docs [#166](https://github.com/nodejs/next-10/issues/166)
+  * Next step was to create demo repository
+  * Claudio walked us through the sample structure, we spent all of the time discussing
+
+We ran out of time and skipped all of the other issues
+* Presentation of the proposal by Claudio [node-doc-proposal](https://github.com/ovflowd/node-doc-proposal)
```
Comment (sheplu) [inline meetings/2023-01-04.md:30]: I think this part should be up - as we spend a good amount of time talking about that too :)

Comment (mhdawson) [reply meetings/2023-01-04.md]: moved

**Review by sheplu — APPROVED** [review]
LGTM

**Pull request merged**